docs/tables/turbot_resource.md

```diff
@@ -106,6 +106,52 @@ where
   and title ilike '%admin%';
 ```
 
+### Search for console logins within 7 days
+
+```sql
+select
+  id,
+  title,
+  data ->> 'email' as email,
+  array_to_string(regexp_matches(trunk_title, '^Turbot > (.*) >'), ' ' ) AS "directory",
+  trunk_title,
+  to_char((data ->> 'lastLoginTimestamp') :: timestamp, 'YYYY-MM-DD HH24:MI') as "last_login"
+from
+  turbot_resource
+where
+  filter = 'resourceTypeId:"tmod:@turbot/turbot-iam#/resource/types/profile" $.lastLoginTimestamp:>=T-7d';
+```
+
+### Search for resources created within 7 days, join with count of controls in alarm state
+
+This query gathers all the recently created resources within 1 day, and joins the resource's current count of controls in alarm state.
```

Comment:
@bob-bot do we still need this description since the query title seems self-explanatory?